.coafile

```diff
@@ -62,7 +62,7 @@ languagetool_disable_rules =
 [commit]
 bears = GitCommitBear
 shortlog_trailing_period = False
-shortlog_regex = ([^:]*|[^:]+[^ ]: [A-Z0-9*].*)
+shortlog_regex = [^:]{3,}|[^:]+[^: ]: [A-Z0-9*].{2,}
```

Comment:
I suggest removing this part of your PR.
The regex is still not very good.  Background reading:
https://www.google.com.au/search?q=good+commit+messages

We can merge your test now.  It is good.